lib/menu_view.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'category_view'
+
+class MenuView
+  MENU = 'MENU'
+  def initialize(menu, category_view_class = CategoryView)
+    @menu = menu
+    @category_view_class = category_view_class
+  end
+
+  def display
+    @menu.categories.reduce("#{MENU}\n") do |sum, category|
+      sum + @category_view_class.new(category).display
```

Comment:
Do you need to instantiate a new instance of CategoryView class for each category, or could you have had a single instance of the class with a 'display' method that takes the category as an argument?